spec/foodzone/venues/list_retriever_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../../../app/foodzone/venues/list_retriever'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.